src/main/java/ua/com/conductor/exception/UsernameNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package ua.com.conductor.exception;
+
+public class UsernameNotFoundException extends RuntimeException {
```

Comment:
you don't need to create your own exception throw already existing in Spring (UsernameNotFoundException) in UserDetailsServiceImpl